src/index.js

```diff
@@ -1,5 +1,5 @@
 import h from "./h"
 import app from "./app"
-import router from "./router"
+import Router from "./router"
```

Comment:
Router could be a plugin but that doesn't mean it has to be. If it is then it negates the `R` vs `r` issue. The awkward thing about the router is that it is not needed for all cases, but in reality it is needed for _a lot_ of cases and if it is not needed (they only have one view/route) then hyperapp already has that covered with very little redundant bytes. Good frontend routers are hard to come by. I think this..
```
model.router = {
	matched: '/some/route/:id',
	url: '/some/route/34593458',
	params: {
		id: 34593458
	}
}
```
with an `onrender` hook, feels like a very robust approach. I'd say for the sakes of argument.. abstract the router out, use it as an example of a really useful plugin (`npm i hyperapp-router -S` is a good practice for developers to get into anyway if we are encouraging a plugin ecosystem) and let people call it what they want.